Backend.Api/Utils/Mongo/MongoDbClientFactory.cs

```diff
@@ -9,7 +9,7 @@ namespace Backend.Api.Utils.Mongo;
 public class MongoDbClientFactory : IMongoDbClientFactory
 {
     private readonly IMongoDatabase _mongoDatabase;
-    private readonly IMongoClient _client;
+   private readonly MongoClient _client;
```

Comment:
is this change intended?

Reply:
SonarCloud moaned about inefficiency using an interface...